unpackaged/232/edaSettings/aura/newEDASettingsQA/newEDASettingsQA.cmp

```diff
@@ -0,0 +1,5 @@
+<aura:component implements="force:appHostable">
+    <aura:attribute name="pageReference" type="Object"/>
+    <aura:handler name="init" value="{!this}" action="{!c.init}" />
+    <c:edaSettings pageReference = "{!v.pageReference}"/>
```

Comment:
We can just use a null page reference here and no other styling. This is a simple wrapper
```suggestion
    <c:edaSettings/>
```